packages/backend/src/netcat.js

```diff
@@ -125,11 +123,19 @@ const inspectPlayersDummy = async () => playersWithDetailsDummy
 const inspectPlayers = NC_DISABLE === 'true' ? inspectPlayersDummy : inspectPlayersReal
 
 const whoIsPlayingReal = async (serverConfig) => {
-  const playerList = [...await nc(serverConfig, [
+  const playerData = await nc(serverConfig, [
     'RefreshList',
```

Comment:
What was the reason for the proposed change?

Reply:
I was getting an issue where if I had no players on my server the backend would crash, citing that `undefined has no attribute .PlayerList`.